docs/components/TemplatesSystem/features/customTags/gimmelist.rst

```diff
@@ -58,6 +59,17 @@ To list articles by authors you can also do:
 
 It will then list all articles written by ``Karen Ruhiger`` **AND** ``Doe``.
 
+To list articles by with source ``Forbes`` but without ``AAP`` you can also do:
```

Comment:
`To list articles from the "Forbes" source but without an "AAP" source you can also do:`